microfrontends/app-builder-menu/src/types/api.ts

```diff
@@ -0,0 +1,33 @@
+export interface MENU_ITEM {
```

Comment:
I'm used to use PascalCase for types and interfaces to avoid using `_` what do u think @Dv92

Reply:
I agree, usually, the notation uppercase snake is for `const`

Reply:
Fixed